README.md

```diff
@@ -1,5 +1,7 @@
 # Facette
 
+:warning: This module is declared as deprecated by Vox Pupuli, see #56. :warning:
```

Comment:
I'd make this a link. Something like
```suggestion
:warning: This module is declared as deprecated by Vox Pupuli, see [#56](https://github.com/voxpupuli/puppet-facette/issues/56). :warning:
```

Reply:
lol, I expected GitHub to do this for me :P